api/src/main/java/org/pmiops/workbench/institution/InstitutionMapper.java

```diff
@@ -20,6 +20,8 @@
   @Mapping(target = "institutionId", ignore = true)
   DbInstitution modelToDb(Institution modelObject);
 
+  // userInstructions will be populated by InstitutionUserInstructionsMapper
+  @Mapping(target = "userInstructions", ignore = true)
```

Comment:
So the issue was that since FK resides in user instructions, if we include userInstruction in DbInstitution, the save was throwing error and not behaving as it should.

Reply:
the email addrs and domains are like that too - so we should be able to make this work

Reply:
yes thats given me an issue too as its doing insert and not update and adding multiple rows. So as part of the edit institution PR you will be seeing that :)

Reply:
Ripping out userInstructions was done after suggestion from @freemabd

Reply:
I'm not sure I understand yet, but this doesn't need to be a blocker.  We can follow up on Slack.